config-rules.tf

```diff
@@ -1,23 +1,25 @@
 data "template_file" "aws_config_iam_password_policy" {
-  template = "${file("${path.module}/config-policies/iam-password-policy.tpl")}"
+  template = file("${path.module}/config-policies/iam-password-policy.tpl")
 
   vars = {
     # terraform will interpolate boolean as 0/1 and the config parameters expect "true" or "false"
-    password_require_uppercase = "${var.password_require_uppercase ? "true" : "false"}"
-    password_require_lowercase = "${var.password_require_lowercase ? "true" : "false"}"
-    password_require_symbols   = "${var.password_require_symbols ? "true" : "false"}"
-    password_require_numbers   = "${var.password_require_numbers ? "true" : "false"}"
-    password_min_length        = "${var.password_min_length}"
-    password_reuse_prevention  = "${var.password_reuse_prevention}"
-    password_max_age           = "${var.password_max_age}"
+    password_require_uppercase = var.password_require_uppercase ? "true" : "false"
+    password_require_lowercase = var.password_require_lowercase ? "true" : "false"
+    password_require_symbols   = var.password_require_symbols ? "true" : "false"
+    password_require_numbers   = var.password_require_numbers ? "true" : "false"
+    password_min_length        = var.password_min_length
+    password_reuse_prevention  = var.password_reuse_prevention
+    password_max_age           = var.password_max_age
   }
 }
 
 data "template_file" "aws_config_acm_certificate_expiration" {
-  template = "${file("${path.module}/config-policies/acm-certificate-expiration.tpl")}"
+  template = file(
+    "${path.module}/config-policies/acm-certificate-expiration.tpl",
```

Comment:
@exequielrafaela Minor comment: I don't think this formatting is necessary. File function only expects a single path as an argument, so it could used simply as follows:
`file("${path.module}/config-policies/acm-certificate-expiration.tpl")`

Reply:
@diego-ojeda-binbash Thanks for you collab here, update already in place 

- commit: 8dec0a7 fixing file function with single path argument

```bash
╭─delivery at delivery-I7567 in ~/Binbash/repos/BB-Leverage/terraform/terraform-aws-config on eb_terraform_012✘✘✘ using ‹› 19-10-27 - 17:44:44
╰─○ git commit -m "fixing file function with single path argument"  
Check JSON...........................................(no files to check)Skipped
Check for merge conflicts................................................Passed
Check Yaml...........................................(no files to check)Skipped
Detect Private Key.......................................................Passed
Pretty format JSON...................................(no files to check)Skipped
Trim Trailing Whitespace.................................................Passed
markdownlint.........................................(no files to check)Skipped
Terraform docs...........................................................Passed
Terraform fmt............................................................Passed
[eb_terraform_012 8dec0a7] fixing file function with single path argument
 1 file changed, 1 insertion(+), 1 deletion(-)

╭─delivery at delivery-I7567 in ~/Binbash/repos/BB-Leverage/terraform/terraform-aws-config on eb_terraform_012✔ using ‹› 19-10-27 - 17:45:26
╰─○ git push origin eb_terraform_012 
Counting objects: 3, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 322 bytes | 0 bytes/s, done.
Total 3 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
To git@github.com:binbashar/terraform-aws-config.git
   1cfb3bd..8dec0a7  eb_terraform_012 -> eb_terraform_012
```